design/protocol-changes/vote-cert-diffusion.tex

```diff
@@ -136,20 +136,20 @@ \subsection{Certificate diffusion mini-protocol}%
   The client uses protocol-level pipelining to avoid round-trip delays and to make full use of the available bandwidth.
 \item
   As the node is eventually done catching up, its peers run out of certificates to serve, and the node will start sending \msg{MsgRequestObjIdsBlocking} instead.\footnote{
-  In particular, an incomplete/empty reply to \msg{MsgRequestObjIdsBlocking} is analogous to \msg{MsgAwaitReply} in the ChainSync protocol.}
+  In particular, an incomplete/empty reply to \msg{MsgRequestObjIdsNonBlocking} is analogous to \msg{MsgAwaitReply} in the ChainSync protocol.}
```

Comment:
Shouldn't we use the macros `\MsgRequestObjIdsNB` & co instead here?

Reply:
Oh right, I missed these somehow :+1: